docs/react-testing-library/example-intro.mdx

```diff
@@ -4,6 +4,65 @@ title: Example
 sidebar_label: Example
 ---
 
+## Quickstart
+
+This is a very simple setup to get you started. 
+If you want to see a description of what each line does, 
+scroll down to the [annotated version](#quickstart-annotated).
+Scroll down to [Full Example](#full-example) to see a more advanced test setup.
+
+```jsx
+import {render, fireEvent, waitFor, screen} from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Fetch from '../fetch'
+
+test('loads and displays greeting', async () => {
+
+  // ARRANGE
+  render(<Fetch url="/greeting" />)
+  
+  // ACT
+  fireEvent.click(screen.getByText('Load Greeting'))
+  await waitFor(() => screen.getByRole('heading'))
```

Comment:
To follow best practices: 

```suggestion
  await screen.findByRole('heading')
```

Reply:
Is there a reason to use `waitFor` here instead of `findBy`?

Reply:
I guess not. The code snippet was taken from the Full example. Thanks for the feedback.